src/ui/components/delegation/delegateModal/Delegate.tsx

```diff
@@ -59,43 +97,63 @@ export function DelegateModal({
           <div className="text-left">
             <h2 className="mb-2 text-3xl font-medium">Summary</h2>
             <p className="text-base">
-              You’re about to submit a transaction to delegate your voting power
-              for the following tracks to <b>{name}</b> Delegate.
+              Submitting this transaction will delegate your voting power to
+              <b>{name}</b> Delegate for the following tracks.
             </p>
           </div>
-          <div className="flex flex-col gap-1">
-            <div className="text-sm">Your delegate</div>
-            <div className="flex gap-2">
-              <Accounticon
-                textClassName="font-medium"
-                address={address}
-                size={24}
-              />
-              <div className="capitalize">{name}</div>
-            </div>
+          <div className="grid w-full grid-cols-3 grid-rows-2 gap-4">
+            <LabeledBox className="col-span-2" title="Tracks to delegate">
+              <div>
+                {tracksCaption}
+                {!!remainingCount && (
+                  <>
+                    {' and'} <a>{`${remainingCount} more`}</a>
+                  </>
+                )}
+              </div>
+            </LabeledBox>
+            <LabeledBox title="Tokens to delegate">
+              <div>
+                {(usableBalance &&
+                  formatBalance(usableBalance, { decimals: 12 })) ||
```

Comment:
decimals are hardcoded for Kusama but we need to pull them based on connected network:
I was not sure where is the best place to fetch this but I think it should be added to the state when the state goes to the connected state. 
#293